Gruntfile.js

```diff
@@ -39,9 +39,9 @@ module.exports = function(grunt) {
     'src/renderers/canvas/sigma.canvas.hovers.def.js',
     'src/renderers/canvas/sigma.canvas.nodes.def.js',
     'src/renderers/canvas/sigma.canvas.edges.def.js',
-    'src/renderers/canvas/sigma.canvas.edges.dotCurve.js',
+    'src/renderers/canvas/sigma.canvas.edges.curve.js',
     'src/renderers/canvas/sigma.canvas.edges.arrow.js',
-    'src/renderers/canvas/sigma.canvas.edges.dotCurvedArrow.js',
+    'src/renderers/canvas/sigma.canvas.edges.dotCurve.js',
```

Comment:
currently in the `coreJSFiles` section there is a js for edge hovers curve `src/renderers/canvas/sigma.canvas.edgehovers.curve.js` 

hence it would probably make sense to add the curve js as well `src/renderers/canvas/sigma.canvas.edges.curve.js`